test/conv.lean

```diff
@@ -0,0 +1,58 @@
+/-
+Copyright (c) 2018 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Simon Hudon
```

Comment:
I don't think I'm the author of this file. Maybe Mario?

Reply:
Not me... from the git log it looks like Keeley added these tests in 350e6e2 and fb57843

Reply:
I've put Keeley in.